packages/support/test/unit/env.spec.js

```diff
@@ -36,6 +36,10 @@ describe('env', function () {
     delete process.env.APPIUM_HOME;
 
     env = rewiremock.proxy(() => require('../../lib/env'), overrides);
+
+    env.findAppiumDependencyPackage.cache = new Map();
```

Comment:
should we use LRUCache instead?

Reply:
no, this is just `_.memoize()`, which adds a `cache` prop of type `Map<args, return value>` to the function.  this just ensures the memoization is reset after every test, for defensive purposes